src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -247,6 +248,13 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       System.err.println("Compiler runtime: " + (stop - start) + "ms.");
     }
 
+    try {
+      PrintWriter writer = new PrintWriter(ops.statsfile, "UTF-8");
+      writer.println(Long.toString(stop - start));
```

Comment:
should this be a bit more structured? Maybe some hand crafted json even:
```json
{
  build_time: 123123
}
```
so we could add other examples of this?

Reply:
There's no built in way to process JSON downstream, either from common shell scripts, or via Skylark APIs. What about a simple properties style file and I drop the assumption that the build time appears on a given line?
```
baz=false
build_time= 123123
foo=bar
```
This can be read with the one liner:
```bash
BUILD_TIME=`awk -F '=' '$1 == "build_time" { print $2 }' some_target.statsfile`
```

Reply:
sure. Happy to do the above